src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -338,14 +267,24 @@ $(XamlMetaDataProviderPch)
             File="$(IntDir)mdmerge.rsp" Lines="$(_MdMergeParameters)"
             ContinueOnError="true" Overwrite="true"
             WriteOnlyWhenDifferent="true" />
-        <Copy SourceFiles="@(_MdMergeInputs->'%(WinMDPath)')" DestinationFolder="$(CppWinRTUnmergedDir)\" />
+        <Copy UseHardlinksIfPossible="$(CppWinRTUseHardlinksIfPossible)"
+            SkipUnchangedFiles="$(CppWinRTSkipUnchangedFiles)"
+            SourceFiles="@(_MdMergeInputs->'%(WinMDPath)')"
+            DestinationFolder="$(CppWinRTUnmergedDir)" />
+        <Copy UseHardlinksIfPossible="$(CppWinRTUseHardlinksIfPossible)"
+            SkipUnchangedFiles="$(CppWinRTSkipUnchangedFiles)"
+            SourceFiles="@(_MdMergeReferences->'%(WinMDPath)')" 
+            DestinationFolder="$(CppWinRTMetadataDir)\" />
```

Comment:
ah, and the level of indirection thru CppWinRTMetadataDir with SkipUnchangedFiles - effectively eliminates dupes

Reply:
XML at it's best. :wink:

Reply:
We also only copy the reference instead of using a wildcard into the output folder which elminates dupes as well.